scripts/get-badges.ts

```diff
@@ -56,7 +48,6 @@ interface UsersWithVisitsResult {
     await admin
       .firestore()
       .collection("users")
-      .where("enteredVenueIds", "array-contains-any", venueIdsArray)
```

Comment:
Not sure that this is a change we want to be doing here. The edgecase the changes in this PR was required for is definitely the exception, not the norm. So to have to process so much extra data every time is suboptimal.

This should probably only run when we need this 'special mode' or similar.